starknet_devnet/blocks.py

```diff
@@ -123,6 +123,9 @@ async def generate(
         state_root = DUMMY_STATE_ROOT
         block_number = self.get_number_of_blocks()
         timestamp = state.state.block_info.block_timestamp
+        signatures = []
```

Comment:
Why add these new arguments? Can't they be extracted from `DevnetTransaction`?

Reply:
I came up with something like this ce5e1bfd8e2033b22435ef62429cee21d16bad09